benchmark/inclusion/inclusion.cc

```diff
@@ -480,6 +477,11 @@ namespace aspect
       Vector<float> cellwise_errors_ul2 (this->get_triangulation().n_active_cells());
       Vector<float> cellwise_errors_pl2 (this->get_triangulation().n_active_cells());
 
+      double u_l1;
```

Comment:
Oh, I realize you just copied this from burstedde.cc. Can you clean it up in that file too please?